bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -223,8 +223,6 @@ private func dummy() {
     _ = NSLocalizedString("Page title. Empty", comment: "translators: accessibility text. empty page title.")
     _ = NSLocalizedString("Paste block after", comment: "")
     _ = NSLocalizedString("Paste URL", comment: "")
-    _ = NSLocalizedString("Post content settings", comment: "")
```

Comment:
- Removal
- [Code reference](packages/block-library/src/latest-posts/edit.js#L208:L208)

This string is already translated as it's used in the web version, so we don't need to include it here.